src/app/(tabs)/stats/result/[runningId]/[courseId]/[ghostRunningId].tsx

```
@@ -195,14 +209,24 @@ export default function Result() {
                 value: getFormattedPace(runData?.recordInfo.averagePace ?? 0),
             },
             {
-                description: "케이던스",
-                value: Math.round(runData?.recordInfo.cadence ?? 0),
-                unit: "spm",
+                description: "케이던스(spm)",
+                value:
+                    runData?.recordInfo.cadence ?? 0 > 0
+                        ? Math.round(runData?.recordInfo.cadence ?? 0)
+                        : "--",
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)